test/e2e/setup/global-setup.js

```diff
@@ -50,9 +50,10 @@ const globalSetup = async config => {
   const rpcEndpoint = `${apiOpts.protocol}://${apiOpts.host}:${apiOpts.port}`
 
   await page.context().addInitScript(({ kuboGateway, rpcEndpoint }) => {
-    localStorage.setItem('kuboGateway', JSON.stringify(kuboGateway))
+    localStorage.setItem('kuboGateway', JSON.stringify({ ...kuboGateway, trustlessBlockBrokerConfig: { init: { allowInsecure: true, allowLocal: true } } }))
     localStorage.setItem('ipfsApi', JSON.stringify(rpcEndpoint))
     localStorage.setItem('explore.ipld.gatewayEnabled', 'false') // disable gateway network requests when testing e2e
+    localStorage.removeItem('debug') // this being set breaks redux-bundler...
```

Comment:
we shouldn't need this at all, but playwright likes to take config from my main browser and not reset localStorage config..

Reply:
bah. i think this was because it was set in playwright config: `storageState: 'test/e2e/state.json',`